esmvaltool/cmorizers/data/formatters/datasets/mte.py

```diff
@@ -13,7 +13,6 @@
     Download the following files:
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Call fixed!